<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QOmgDragableEntities</class>
 <widget class="QWidget" name="QOmgDragableEntities">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>100</width>
    <height>857</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>100</width>
    <height>100</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>110</width>
    <height>1677777</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="toolTip">
   <string extracomment="door Enemy"/>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
background-color:#C8C8C8
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="_search">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>20</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>10000</height>
      </size>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="_scroll">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>94</width>
        <height>735</height>
       </rect>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="_counter">
       <property name="text">
        <string>Counter</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="_door_enemy">
     <property name="minimumSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
     <property name="whatsThis">
      <string extracomment="Door Enemy"/>
     </property>
     <property name="text">
      <string>New Door</string>
     </property>
     <property name="icon">
      <iconset resource="../../OmegaEditor.qrc">
       <normaloff>:/OmegaEditor/Resources/Icons/enemy.png</normaloff>:/OmegaEditor/Resources/Icons/enemy.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="_door_player">
     <property name="minimumSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
     <property name="whatsThis">
      <string extracomment="Door Enemy"/>
     </property>
     <property name="text">
      <string>New Door</string>
     </property>
     <property name="icon">
      <iconset resource="../../OmegaEditor.qrc">
       <normaloff>:/OmegaEditor/Resources/Icons/character.png</normaloff>:/OmegaEditor/Resources/Icons/character.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../OmegaEditor.qrc"/>
  <include location="../../OmegaEditor.qrc"/>
 </resources>
 <connections/>
</ui>
